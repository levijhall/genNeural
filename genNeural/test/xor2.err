Training stopped 15122 / 500000 (3.02%)
Avg. error = 0.001 < 0.001
[1.000 0.000 0.000 ] | [0.020 ] - [0.000 ] (0.000)
[1.000 1.000 1.000 ] | [0.982 ] - [1.000 ] (0.000)
[1.000 1.000 1.000 ] | [0.982 ] - [1.000 ] (0.000)
[0.000 0.000 0.000 ] | [0.032 ] - [0.000 ] (0.001)
