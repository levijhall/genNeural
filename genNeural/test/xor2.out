7
1 4 2.583943
2 4 0.887924
3 4 0.864109
1 5 -1.821174
2 5 2.007554
3 5 2.223188
1 6 -1.181881
2 6 1.259571
3 6 1.755561
1 7 5.467033
4 7 -7.637327
5 7 3.134163
6 7 1.081029
